Test/git-issues/git-issue-731b.dfy

```diff
@@ -0,0 +1,24 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
+// RUN: %dafny /noVerify /compile:4 /compileTarget:cs "%s" >> "%t"
+// RUN: %dafny /noVerify /compile:4 /compileTarget:js "%s" >> "%t"
+// RUN: %dafny /noVerify /compile:4 /compileTarget:go "%s" >> "%t"
+// RUN: %dafny /noVerify /compile:4 /compileTarget:java "%s" >> "%t"
+// RUN: %diff "%s.expect" "%t"
+
+// Testing issue#731 when the class in question has type parameters
+
+trait Tr2<W,Y> {
+  const w: W;
```

Comment:
```suggestion
  const w: W
```

Reply:
When I don't type semicolons reflexively (and don't put types before variables), I'll really be a Dafny programmer